### ISSUE TITLE
timescale for including MPL-7?

### ISSUE THREAD

**Issue by aleksds** [sdss/marvin#417]
This is not a pressing concern, but I was just curious about the planned timeframe for including MPL-7 as a valid release for Marvin. My specific thought is motivated by work with students who have downloaded the MPL-7 maps locally.

**Comment by albireox** [comment]
Hi @aleksds,

Right now I don't think there is a timeframe, @havok2063 probably has a better feeling on when this will be possible, given the migration to new servers at Utah.

I think it should be relatively simple to allow Marvin to use MPL-7 local files (no web, no queries), if that is what you are interested in. I'll have a look and see if it's possible to enable that easily.

**Comment by havok2063** [comment]
We decided not to release MPL-7 in Marvin, per discussions on the lists and the DevCon and PipeCons back in March.  This was originally due to MPL-7 not being in the timeline for Marvin up to DR15 along with uncertainties about the hardware configuration.  That's primarily why there has been no movement on it.   The plan was to proceed along the original timeline and have the next Marvin release be DR15 which is MPL-7.       

The new hardware is set up now though and I've loaded the MPL-7/DR15 data into the database.  I need to check that everything is kosher with it.  If we wanted to have a separate mini-release of MPL-7 we would need to cherry pick out the MPL-7 datamodel additions in the `dr15` branch and merge them into `master` and make a new tag.  I **think** that's all we'd need to do.
